lib/rules/jsx-handler-names.js

```diff
@@ -21,7 +21,7 @@ module.exports = {
   meta: {
     docs: {
       description: 'Enforce event handler naming conventions in JSX',
-      category: 'Stylistic Issues',
+      category: 'Best Practices',
```

Comment:
i'm not sure this really qualifies as a best practice; stylistic feels appropriate here